### PR TITLE
Bg user story 33

### LINKED ISSUE #36: User Story 33, Admin can "ship" an order
As an admin user
When I log into my dashboard, "/admin"
Then I see any "packaged" orders ready to ship.
Next to each order I see a button to "ship" the order.
When I click that button for an order, the status of that order changes to "shipped"
And the user can no longer "cancel" the order.

### PR DESCRIPTION
**Pull request by brian-greeson**
## Admin can ship order
Closes: #36 
- Added `ship` method to Order model
- Added guard clauses to `ship` and `cancel` Order methods to ensure that 
   - A) a canceled order cannot be shipped
   - B) a shipped order cannot be canceled

### CODE REVIEW

File: app/models/order.rb [hunk 1]
```diff
@@ -14,10 +14,16 @@ def total_items
...
   def cancel
+    return false if status == "shipped"
     item_orders.each do |item_order|
       item_order.unfulfill
     end
     self.update(status: "canceled")
   end
+
+  def ship
+    return false if item_orders.where(status: "unfulfilled").any?
```
Comment (fredrondina) [inline app/models/order.rb:25]: nice guard here

File: app/views/admin/dashboard/index.html.erb [hunk 2]
```diff
@@ -1,51 +1,60 @@
 <h1>Admin Dashboard</h1>
-
-<section class="packaged_orders">
+save_and_open_page
```
Comment (fredrondina) [inline app/views/admin/dashboard/index.html.erb]: get this removed and then we are good to go

**Review by fredrondina — CHANGES_REQUESTED** [review]
Looks great, however need to remove the save and open page

File: app/views/admin/dashboard/index.html.erb (hunk 2 above)
Comment (brian-greeson) [reply app/views/admin/dashboard/index.html.erb]: done

**Review by fredrondina — APPROVED** [review]
Great work

**Review by kelshadarby — APPROVED** [review]
This looks awesome. Nice work!

**Pull request merged**